core/services/ocr3/promwrapper/plugin_test.go

```diff
@@ -132,7 +144,7 @@ func (f fakePlugin[RI]) Outcome(context.Context, ocr3types.OutcomeContext, ocrty
 	if f.err != nil {
 		return nil, f.err
 	}
-	return ocr3types.Outcome{}, nil
+	return make([]byte, 3), nil
```

Comment:
can we use variable for the size for clarity? so the assertion earlier will be `10*outcomeSize` for example